_includes/nav_list

```diff
@@ -7,10 +7,13 @@
   <ul class="nav__items">
     {% for nav in navigation %}
       <li>
+        {% if nav.icon %}
+            {% capture icon %}<i class="{{ nav.icon }}" aria-hidden="true"></i> {% endcapture %}
+        {% endif %}
         {% if nav.url %}
-          <a href="{{ nav.url | relative_url }}"><span class="nav__sub-title">{{ nav.title }}</span></a>
```

Comment:
Shouldn't this keep the relative_url filter?

Reply:
This `nav_url` comes from nowhere - am I mistaken?

Reply:
Honestly I don't remember it and it puzzles me a lot.